### PR TITLE
content updated

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,27 +1,40 @@
-= An Example Quick Course
+= Getting Started with Red Hat Connectivity Link
 :navtitle: Home
 
 == Introduction
 
-Course Title: Course title FIXME
+Welcome to this quick course on the _Getting Started with Red Hat Connectivity Link_.
+This course is the `second` in a series of `two` courses about Red Hat Connectivity Link.
```
Comment (rsriniva) [inline modules/ROOT/pages/index.adoc:7]: List both names in order here with links.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
-Description:
-Course description FIXME
+In this hands-on lab, you will learn how to secure, protect, and connect an API through a Gateway based on the Kubernetes Gateway API, all within a single OpenShift cluster with a shared HTTPS listener. 
 
-Duration: FIXME hours
+This lab is designed to use Red Hat Connectivity Link (RHCL) with a specific use case to implement its features, focusing on two different user roles:
+
+* **Platform Engineer**: As a Platform Engineer, this lab will guide you through deploying a Gateway that ensures secure communication, making it ready for application development teams. You'll learn to route specific traffic while maintaining security through global rate limiting and authentication.
+
+* **Application Developer**: As an Application Developer, this lab will show you how to deploy your application API and override the Gateway's global authentication and rate limiting policies to tailor them to your application’s specific needs.
```
Comment (rsriniva) [inline modules/ROOT/pages/index.adoc:15]: Are you really overriding the rate limits and auth at the app level? check with with Jaya

File: modules/chapter1/pages/index.adoc [hunk 3]
```diff
@@ -1,7 +1,10 @@
-= Chapter 1
+= Overview of Red Hat Connectivity Link (RHCL)
 
-In this section, we cover... FIXME
+The objective of this chapter is to provide a comprehensive overview of Red Hat Connectivity Link (RHCL), an open-source solution tailored for hybrid cloud environments.
```
Comment (rsriniva) [inline modules/chapter1/pages/index.adoc:3]: "open-source solution tailored for hybrid cloud environments" is too generic

File: modules/chapter1/pages/section1.adoc [hunk 4]
```diff
@@ -1,6 +1,15 @@
-= Section 1
+= What is Red Hat Connectivity Link (RHCL)?
+
+* Red Hat Connectivity Link (RHCL) is a cloud-native, open-source solution designed for hybrid cloud environments.
```
Comment (rsriniva) [inline modules/chapter1/pages/section1.adoc:3]: too generic. too many things are "cloud-native, open-source solution for OHC" - Be specific

File: modules/chapter2/pages/section1.adoc [hunk 5]
```diff
@@ -1,6 +1,30 @@
...
+
+In this lab exercise, you will configure Amazon Route 53 to manage DNS and direct traffic to your AWS resources effectively.
+
+. Access the AWS Lab Environment:
+* Log in to the newly created AWS lab environment using the provided credentials in your lab environment page.
+
+. Follow the Video Instructions:
+* The video provides a step-by-step guide to setting up Route 53. It covers creating a hosted zone, configuring DNS records, and associating Route 53 with an existing Virtual Private Cloud (VPC).
++
+video::route53.mp4[align="center",width=640,height=480]
```
Comment (rsriniva) [inline modules/chapter2/pages/section1.adoc:11]: video not loading or playing

Comment (rsriniva) [reply modules/chapter2/pages/section1.adoc:11]: check RHOAI or Dev Hub intro course asciidoc. videos need to be copied in a specific named folder

**Review by rsriniva — APPROVED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (YogiSoni-lazy) [reply modules/ROOT/pages/index.adoc:7]: added the link

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/ROOT/pages/index.adoc:15]: corrected the wordings to:
As an Application Developer, this lab will show you how to deploy the a **toystore** application, configure external access via an HTTPRoute, and secure it using Kuadrant policies for authentication and access control.

File: modules/chapter2/pages/section1.adoc (hunk 5 above)
Comment (YogiSoni-lazy) [reply modules/chapter2/pages/section1.adoc:11]: corrected the issues, thanks for course references

**Pull request merged**